client/packages/system/src/Stock/Components/Ledger/LedgerForm.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, useState } from 'react';
+import {
+  useTranslation,
+  Box,
+  TableProvider,
+  createTableStore,
+  DataTable,
+  SortBy,
+} from '@openmsupply-client/common';
+import { LedgerRowFragment, StockLineRowFragment } from '../../api';
+import { useStockLedger } from '../../api/hooks/useStockLedger';
+import { ColumnKey, useLedgerColumns } from './useLedgerColumns';
+
+interface LedgerFormProps {
+  stockLine: StockLineRowFragment;
+}
+export const LedgerForm: FC<LedgerFormProps> = ({ stockLine }) => {
+  const t = useTranslation('inventory');
+
+  const [sortBy, setSortBy] = useState<SortBy<LedgerRowFragment>>({
+    key: ColumnKey.DateTime,
+    direction: 'desc',
+  });
+  const { data, isLoading, isError } = useStockLedger(stockLine, sortBy);
+  const { columns } = useLedgerColumns(sortBy, (key, direction) => {
+    setSortBy({ key, direction });
+  });
+
+  return (
+    <Box display="flex" sx={{ overflowY: 'auto' }}>
```

Comment:
Table isn't scrolling, does it need max height? ... I think we should always be able to see bottom modal buttons?
![Screenshot 2024-04-19 at 3 20 07 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/efa39349-6a45-4fa3-84aa-732328631edf)

Reply:
Yes: cbd8b0f14